.github/workflows/conductor-complex-scenarios-tests.yml

```diff
@@ -80,7 +80,7 @@ jobs:
         bash ./docker.local/bin/start.conductor.sh view-change.fault-tolerance
 
     - name: Running No VC Fault Tolerance Tests.
-      if: ${{ github.event.inputs.view_change == 'true' }}
```

Comment:
why do we need this input variable? We want to run this test on each update.
and below, lines 149, 154?